test/js-specs.js

```diff
@@ -142,12 +142,32 @@ describe('Advanced HTTP public interface', function () {
   it('configures global timeout value correctly with given valid value', () => {
     http.setRequestTimeout(10);
     http.getRequestTimeout().should.equal(10);
+    http.getConnectTimeout().should.equal(10);
+    http.getReadTimeout().should.equal(10);
   });
 
   it('throws an Error when you try to configure global timeout with a string', () => {
     (() => { http.setRequestTimeout('myString'); }).should.throw(messages.INVALID_TIMEOUT_VALUE);
   });
 
+  it('configures connect timeout and read timeout with given valid value', () => {
```

Comment:
Actually you do not test for read timeout here, did you mean something like "configures connect timeout value correctly with given valid value"?

Reply:
Yes, agreed